kratos/python/add_utilities_to_python.cpp

```diff
@@ -260,16 +260,11 @@ void AddUtilitiesToPython(pybind11::module& m)
     .def("FindMaximumEdgeSize", &ParallelDistanceCalculator < 3 > ::FindMaximumEdgeSize)
     ;
 
-    class_<PointLocation >(m,"PointLocation")
+    class_<PointLocator> (m, "PointLocator")
```

Comment:
why did u change the name?

Reply:
1. to make clear that this is a complete (and more general) reimplementation

2. the new name makes more sense to me -> this calss locates points, it is not itself a location

Reply:
I also prefer the new name.

Reply:
but still, by changing the name you are breaking previous usages. that s not a good way to go ...  (i agree the name is better)

Reply:
It is not used anywhere else in the code except in the files I modify in this PR

If you prefer the old one I can change it back, you call :)

Reply:
No, if it is not used anywhere (including python) then you can change it.

However I would propose to take the occasion to call it something like
**BruteForcePointLocator**

Reply:
would u agree naming it "BruteForcePointLocator" or smthg of the sort?

Reply:
Sounds good, will change it